faq.md

```diff
@@ -233,10 +232,10 @@ The primary criteria we used to pick a programming language were the following:
   performance and delivers predictable performance, without periodic
   pauses that would cause dropped frames.
 
-* _Fast allocation_. The Flutter framework uses a functional-reactive style
+* _Fast allocation_. The Flutter framework uses a reactive style
   programming model, whose performance depends heavily on the underlying memory
   allocator efficiently handling small, short-lived allocations. The
-  functional-reactive style was developed in languages with this property and
+  reactive style was developed in languages with this property and
```

Comment:
Is this statement still accurate?

The "functional-reactive" style was developed in different languages than the "reactive" style. I would be wary of changing this statement without verifying the accuracy of the new claim.